packages/foam-cli/src/utils/rename-file.ts

```diff
@@ -0,0 +1,11 @@
+import * as fs from 'fs'
+
+
+
+export const renameFile = async (fileUri: string, newFileName: string) => {
+  const fileName = fileUri.split('/').pop();
+  const extension = fileName?.split('.').pop();
+  const newFileUri = fileUri.replace(`${fileName}`, `${newFileName}.${extension}`);
```

Comment:
This would break if the file name is anywhere in the directory, e.g. `markdown/markdown.md`.

Reply:
It should be fixed now.